lib/content._coffee

```diff
@@ -6,21 +6,18 @@ redis = require 'redis'
 URL = require 'url'
 Worker = require './worker'
 
+pkgcloud = require('pkgcloud')
+cloudClient = pkgcloud.storage.createClient
```

Comment:
Maybe call this `cloudFiles` or `storageClient` instead of `cloudClient`, since it's specific to files/storage.